### PR TITLE
Created the Landing Page

### PR DESCRIPTION
**Pull request by Sharmaa-Ansh**

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="https://iconica.netlify.app/Components.css">
+    <link rel="stylesheet" href="/Index.css">
     <title>Document</title>
```
Comment (ravikumar1002) [inline Index.html:9]: change title

Comment (Sharmaa-Ansh) [reply Index.html:9]: noted

File: Index.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+footer{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    font-size: 1.2rem;
+    padding: 1rem 0;
+    background-color: var(--secondary-color);
+    color: white;
```
Comment (ravikumar1002) [inline Index.css:74]: use color variable

**Review by ravikumar1002 — APPROVED** [review]

**Comment by ravikumar1002** [comment]
looks good to merge

**Review by Devansu-Yadav — APPROVED** [review]
Well-written CSS code and good use of BEM convention. Looks good to merge. Just one small suggestion - You can enable the PR preview feature of Netlify for PR reviewers to actually see the webpage before you merge.

**Pull request merged**